### DISCUSSION TITLE
Quais plugins vocês usam no Vscode?

### DISCUSSION THREAD

**Discussion by deppbrazil** [frontendbr/forum#1363] [Perguntas e Respostas]
## Fala galera blz? ##

Queria fazer uma pesquisa rápida aqui sobre quais plugins vocês têm usado no VSCode no dia-a-dia e por que?

**Comment by luizbills** [comment]
Quando eu usava, só lembro de ter instalado "EditorConfig" e alguns de syntax highlighting de algumas tecnologias. Acho que ele já vem com "Emmet" que é outro que eu gosto de instalar.

**Comment by williangd** [comment]
EditorConfig
Prettier
GitLens
TSLint
ESLint
Bracket Pair Colorizer
Docker

**Comment by deppbrazil** [comment]
Eu instalei alguns desse [artigo](https://willianjusten.com.br/migrei-para-o-vscode-e-estou-feliz/) do Willian Justen

**Comment by feliperom** [comment]
Beautify
EditorConfig
Eslint
TSLint
HTML Snippets
JavaScript (ES6) code snippets
IntelliSense for CSS
Sass
React Native Tools
ES7 React/Redux/GraphQL/React-Native snippets

**Comment by SrVieira** [comment]
Utilizo os seguintes plugins, tornam o desenvolvimento muito mais agradável

Sass
Prettier
Git History
Docker
ESLint
vscode-icons
ES7 React/Redux/GraphQL/React-Native snippets
import costs (esse é top para para termos uma ideia do custo de cada importação de dependencia/component)

**Comment by tcelestino** [comment]
Prettier
ESLint
vscode-icons
Docker
Auto Close Tag
Auto Rename Tag
Bracket Pair Colorizer
Dash (https://kapeli.com/dash)
DotENV
Files Utils
Javascript (ES6) code snippets
npm
npm Intellisense
Path Intellisense
Settings Sync (essa eu super recomendo!! você não vai mais precisar se preocupar com as configurações do VSCode em qualquer ambiente (linux, mac, windows))
Waka Time

Extra: One Monokai Theme (nunca mais troquei de tema - :smile:)

**Comment by romulosardinha** [comment]
bernardodsanderson.theme-material-neutral
formulahendry.auto-close-tag
formulahendry.auto-rename-tag
HookyQR.minify
ionutvmi.path-autocomplete
liximomo.sftp
OfHumanBondage.react-proptypes-intellisense
PKief.material-icon-theme
ritwickdey.live-sass
ritwickdey.LiveServer
skacekachna.win-opacity
teamchilla.blueprint
WakaTime.vscode-wakatime

**Comment by romulosardinha** [comment]
@tcelestino eu tbm estava usando o Settings Sync, mas ultimamento não está sincronizando, ainda não parei para ver o que é.

**Comment by thiagofrancisquete** [comment]
Como _"sou do angular"_ uso esse pack da Loiane https://marketplace.visualstudio.com/items?itemName=loiane.angular-extension-pack

**Comment by NicholasKuchiniski** [comment]
Para a galera que gosta de temas, super recomendo: https://github.com/rainglow/vscode
Tem temas infinitos, uso a um tempo e até hoje não consegui ver todos.

**Comment by yuritoledo** [comment]
Bookmarks - para "favoritar" linhas do projeto.
Bracket Pair Colorizer
change-case
css-in-js
debugger for chrome
ES7 react/redux{...}
ESLint
GitLens
jumpy (esse é ótimo pra parar de usar as "setinhas" pra navegar no código)
markdown all in one
material theme
react native tools
settings sync
svg viewer
turbo console log

**Comment by johanguse** [comment]
Todos esses que a galera citou!
Dou um plus para o **Settings Sync**
